eventq_aws/lib/eventq_aws/aws_queue_worker.rb

```diff
@@ -33,7 +33,7 @@ def start(queue, options = {}, &block)
         configure(queue, options)
 
         if options[:client] == nil
```

Comment:
as per comment on this if statement within the rabbitmq implementation of the worker below, I suggest we modify this if statement to verify either `options[:client] == nil || options[:aws_account_no]` is specified else raise error.